about.html

```diff
@@ -59,7 +59,7 @@
 
       <div class="desktop-navbar">
```

Comment:
- [ ] Your about page destop view should have a dark background footer, kindly fix this
  
![Screenshot from 2022-08-02 22-51-15](https://user-images.githubusercontent.com/66097371/182480144-f3c8ae8f-52f6-4dc9-bc6e-1959cf5943f7.png)